app/helpers/application_helper.rb

```diff
@@ -577,4 +577,33 @@ def radio_button_list(form, field_name, option_array)
       end
     end
   end
+
+  # Identifier for creation, formatted external-work-12, series-12, work-12.
+  def creation_id_for_css_classes(creation)
+    return if creation.nil?
+    return unless %w[ExternalWork Series Work].include?(creation.class.name)
+
+    "#{creation.class.name.underscore.dasherize}-#{creation.id}"
+  end
+
+  # Space-separated list of creator ids, formatted user-123 user-126.
+  # External works are not created by users, so we can skip this.
+  # TODO: We will want to list users on series blurbs, but when a series becomes
```

Comment:
Separate, I think. It's not about adding an anonymous work to a series (since that won't change the series to being anonymous), but rather

- post a single work in a series
- go back and add that work to an anonymous (or unrevealed) collection

When that happens, we'll want to remove the user info from the blurb's CSS, which means we'll need to expire the cached class names.

Reply:
Should we have an issue for that?

Reply:
I see you made https://otwarchive.atlassian.net/browse/AO3-6132, thanks!